### PR TITLE
npm→bun

### PR DESCRIPTION
**Pull request by merchako**
Use Bun instead of NPM. It's a lot faster.
requires [installation of bun.sh](https://bun.sh/docs/installation)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
     "eslint-plugin-svelte3": "^3.2.1",
     "lodash-es": "^4.17.21",
     "material-components-web": "^14.0.0",
-    "npm-run-all": "^4.1.5",
+    "bun-run-all": "^4.1.5",
```
Comment (hobbitronics) [inline package.json:47]: I don't think this will work

**Review by hobbitronics — CHANGES_REQUESTED** [review]
We need to make sure all the commands work. I haven't been able to get docker to work. I think it would be nice to use bun-bundler.

File: package.json (hunk 1 above)
Comment (merchako) [reply package.json:47]: oh whoops!

Comment (merchako) [reply package.json]: Resolved in [fd3bf81](https://github.com/silinternational/cover-ui/pull/612/commits/fd3bf817ce7c0470f63e7aa8ac3cb54e3b14d7c7)

File: package.json [hunk 2]
```diff
@@ -5,18 +5,18 @@
...
     "clean": "run-p -l clean:build clean:deps",
     "clean:build": "rimraf dist/bundle* dist/workbox*",
-    "clean:deps": "npm prune",
+    "clean:deps": "bun prune",
     "format": "prettier --write ./src/**/*.{js,ts,svelte}",
     "lint": "eslint ./src/**/*.{js,ts,svelte}",
     "lint:fix": "eslint --fix ./'src/**/*.{js,ts,svelte}'",
-    "prelint": "npm run format",
+    "prelint": "bun run format",
     "local:dev": "run-p -l build:dev serve:http",
```
Comment (hobbitronics) [inline package.json:20]:
```suggestion
     "local:dev": "conc \"bun run build:dev\" \"bun run serve:http\"",
```

File: package.json [hunk 3]
```diff
@@ -5,18 +5,18 @@
...
     "local:prod": "run-p -l build:prod serve:http",
```
[the remark this answers is not in the document]
Comment (hobbitronics) [reply package.json:21]: You'll need `npm i -D concurrently`

**Comment by briskt** [comment]
I think we need to remove bun.lockb from the repo and add it to .gitignore

**Comment by hobbitronics** [comment]
I've merged the other bun PR so closing this. Hopefully bun will add more compatibility so we can use it in Docker with Rollup.

**Pull request closed without merging**